vendor/pybind11/.appveyor.yml

```diff
@@ -0,0 +1,66 @@
+version: 1.0.{build}
```

Comment:
I would only include the files from pybind11 that we actually need:
- the necessary headers
- the `CMakeLists.txt` file (and whatever it requires)
- the license